share/efuncs.sh

```diff
@@ -1338,6 +1338,76 @@ directory_not_empty()
     find "${dir}" -mindepth 1 -print -quit | grep -q .
 }
 
+opt_usage relative_path<<'END'
+# relative_path takes two paths and figures out the relative path between the two.
+# ex: the relative_path between /home/user/foo/bar and /home/user/baz/buz is
+# ../../baz/buz
+END
+relative_path()
+{
+  $(opt_parse \
+    "start  | Starting directory" \
+    "target | Target directory")
+
+  edebug "${LINENO}: $(lval start target)"
```

Comment:
edebug already emits the file, function and line number, so ${LINENO} isn't necessary here.

Reply:
:)  these were einfos when I was using them, however you're right, this would need to be changed if we were keeping it